blog_post.md

```diff
@@ -1,52 +1,290 @@
+spec:
+  bindAsFiles: false
+  services:
+  - group: rabbitmq.com
+    version: v1beta1
+    kind: RabbitmqCluster
+    name: rabbitmq
+  application:
+    name: consumer-deployment
+    version: v1
+    group: apps
+    resource: deployments
+---
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: servicebinding-producer
+spec:
+  bindAsFiles: false
+  services:
+  - group: rabbitmq.com
+    version: v1beta1
+    kind: RabbitmqCluster
+    name: rabbitmq
+  application:
+    name: producer-deployment
+    version: v1
+    group: apps
+    resource: deployments
+---
+```
 
-Instead of doing this, we can instead leverage SBO to get these services to talk
-to RabbitMQ.
+Now, if we inspect the logs of our `consumer` deployment, we'll see that we've
+been receiving messages from our `producer`.  You should see something similar
+to the following:
+```
+$ kubectl logs consumer-deployment-6f48dbfb7d-5dsgd
+connecting to: amqp://default_user_7Jba_ZP7NKD-UjJK8AQ:HIhVZ4a_6Xm60Z7bmbEDADDpwr2e_tch@rabbitmq.default.svc:5672
+Waiting for messages, press Ctrl-C to exit.
+(  0) Received [hello, world!]
+(  1) Received [hello, world!]
+(  2) Received [hello, world!]
+(  3) Received [hello, world!]
+(  4) Received [hello, world!]
+(  5) Received [hello, world!]
+(  6) Received [hello, world!]
+(  7) Received [hello, world!]
+(  8) Received [hello, world!]
+(  9) Received [hello, world!]
+( 10) Received [hello, world!]
+( 11) Received [hello, world!]
+( 12) Received [hello, world!]
+```
 
-TODO: detail [jobs.yaml](/jobs.yaml) and [service-binding.yaml](/service-binding.yaml)
+`producer` says something similar:
 
-## An even easier way
+```
+kubectl logs producer-deployment-6d8d55949d-8qd9c
+connecting to: amqp://default_user_7Jba_ZP7NKD-UjJK8AQ:HIhVZ4a_6Xm60Z7bmbEDADDpwr2e_tch@rabbitmq.default.svc:5672
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+sending [hello, world!] to queue hello
+```
 
-Set the label `service.binding/provisioned-service=true` on the custom resource
-(instead of the annotations we would usually set) and everything should work.
+# An even easier way
 
-TODO: (insert details of it working here)
+We don't always need to specify these annotations on our custom resources.
+Instead, we could instead set the label `servicebinding.io/provisioned-service:
+true` on the custom resource (instead of the annotations we would usually set)
+and everything should work.  Ideally, this would already be done for us;
+however, at time of writing, this label has not been applied to RabbitMQ's
+operator.
```

Comment:
this annotation is applied out-of-the-box by SBO RabbitMq support.

Reply:
Right, I was misreading the spec.  It's *recommended* that provisioned services put this label in, but it's not *required* that they do so.  I'll take this out.